src/storage/mvcc/mod.rs

```diff
@@ -1,6 +1,111 @@
 // Copyright 2016 TiKV Project Authors. Licensed under Apache-2.0.
 
 //! Multi-version concurrency control functionality.
+//!
+//! The module encapsulates the TiKV's MVCC operations.
+//!
+//! # Structure
+//!
+//! There are 3 submodules:
+//!
+//! `reader` module implements read operations, including scanning.
+//!
+//! `metrics` module includes Prometheus metrics.
+//!
+//! `txn` module implements MVCC transaction operations, e.g. commit and rollback.
```

Comment:
I think it's not necessary to describe the structure in the module doc. Maybe it's better to make these description the documentation of each submodule.